scripts/us_eia/eia_860/power_plant.py

```diff
@@ -0,0 +1,241 @@
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Processor for Schedule 2 of EIA Form 860 which includes Power Plant data.
+
+To run this script as a standalone:
+python3 -m power_plant <IN_EXCEL_FILENAME> <OUT_CSV_FILENAME>
+"""
+import csv
+import io
+import os
+import sys
+
+import pandas as pd
+import pandas.api.types as pd_types
+import numpy as np
+import typing
+
+import utils
+
+IMPORT_COLUMNS = [
+    'UtilityId',
+    'UtilityName',
+    'PlantCode',
+    'PlantName',
+    'StreetAddress',
+    'City',
+    'State',
+    'Zip',
+    'County',
+    'Latitude',
+    'Longitude',
+    'NERC Region',  # Not imported
+    'Balancing Authority Code',  # Not imported
+    'Balancing Authority Name',  # Not imported
+    'Name of Water Source',  # Not imported
+    'NAICSCode',
+    'RegulatoryStatus',
+    'Sector',
+    'Sector Name',
+    'FERC Cogeneration Status',
+    'FercCogenerationDocketNumber',
+    'FERC Small Power Producer Status',
+    'FercSmallPowerProducerDocketNumber',
+    'FERC Exempt Wholesale Generator Status',
+    'FercExemptWholesaleGeneratorDocketNumber',
+    'Ash Impoundment?',
+    'Ash Impoundment Lined?',
+    'Ash Impoundment Status',
+    'Transmission or Distribution System Owner',  # Not imported
+    'Transmission or Distribution System Owner ID',  # Not imported
+    'Transmission or Distribution System Owner State',  # Not imported
+    'Grid Voltage 1',
+    'Grid Voltage 2',
+    'Grid Voltage 3',
+    'Energy Storage',
+    'Natural Gas LDC Name',  # Not imported
+    'Natural Gas Pipeline Name 1',  # Not imported
+    'Natural Gas Pipeline Name 2',  # Not imported
+    'Natural Gas Pipeline Name 3',  # Not imported
+    'Pipeline Notes',  # Not imported
+    'Natural Gas Storage',
+    'Liquefied Natural Gas Storage',
+]
+
+EXPORT_COLUMNS = [
+    'Dcid',
+    'Name',
+    'Address',
+    'StateDcid',
+    'ZipDcid',
+    'Latitude',
+    'Longitude',
+    'Naics',
+    'PlantCode',
+    'UtilityDcid',
+    'RegulatoryStatusEnum',
+    'PowerPlantSector',
+    'FercCogenerationStatus',
+    'FercCogenerationDocketNumber',
+    'FercSmallPowerProducerStatus',
+    'FercSmallPowerProducerDocketNumber',
+    'FercExemptWholesaleGeneratorStatus',
+    'FercExemptWholesaleGeneratorDocketNumber',
+    'IsAshImpounded',
+    'IsAshLined',
+    'AshImpoundmentStatusEnum',
+    'GridVoltage1',
+    'GridVoltage2',
+    'GridVoltage3',
+    'IsEnergyStored',
+    'IsNaturalGasStored',
+    'IsLiquefiedNaturalGasStored',
+]
+
+REGULATORY_STATUS_ENUM = {
+    'NR': 'dcid:Regulated',
+    'RE': 'dcid:Non-regulated',
```

Comment:
NonRegulated ?

Reply:
done (but kept the name as "Non-regulated"